<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2P Free Market</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>30</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTableView" name="account_table">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QTableView" name="content_table">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="trx_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
    <addaction name="actionNew_Account"/>
    <addaction name="actionTransfer"/>
    <addaction name="actionIssue"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuNode">
    <property name="title">
     <string>Node</string>
    </property>
    <addaction name="actionGenerate_Blocks"/>
    <addaction name="separator"/>
    <addaction name="actionAnother_Command"/>
    <addaction name="actionAnd_Again"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
   </widget>
   <addaction name="menuAccount"/>
   <addaction name="menuNode"/>
   <addaction name="menuNetwork"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Account">
   <property name="text">
    <string>New Account</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionTransfer">
   <property name="text">
    <string>New Transaction</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionGenerate_Blocks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionConfigure_Generation">
   <property name="text">
    <string>Configure Generation</string>
   </property>
  </action>
  <action name="actionIssue">
   <property name="text">
    <string>Issue Stock</string>
   </property>
  </action>
  <action name="actionAnother_Command">
   <property name="text">
    <string>Con Gen</string>
   </property>
  </action>
  <action name="actionAnd_Again">
   <property name="text">
    <string>And Again</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
